js_modules/dagit/packages/core/src/resources/ResourceRoot.tsx

```diff
@@ -238,9 +316,33 @@ const RESOURCE_ROOT_QUERY = gql`
           value
           type
         }
+        nestedResources {
+          name
+          resource {
+            name
+            resourceType
+          }
+        }
+        resourceType
       }
       ...PythonErrorFragment
     }
   }
   ${PYTHON_ERROR_FRAGMENT}
 `;
+
+const SECTION_HEADER_HEIGHT = 48;
+const SectionHeader = styled.div`
+  background-color: ${Colors.Gray50};
+  border: 0;
+  box-shadow: inset 0px -1px 0 ${Colors.KeylineGray}, inset 0px 1px 0 ${Colors.KeylineGray};
+  cursor: pointer;
```

Comment:
Does the `SectionHeader` have any interactive behavior? If not, it should be fine to leave the cursor as the default.